community_supplied/Linecard/netconf-fpc-cpu-stats.rule

```diff
@@ -0,0 +1,261 @@
+                }
+                type string;
+                description "fpc slot number";
+            }
+            field threshold {
+                constant {
+                    value "{{static-threshold}}";
+                }
+                type integer;
+                description "Threshold value for memory & CPU utilization values";
+            }
+			/*
+            * Anomaly detection logic for buffer memory utilization.
+            */
+            trigger buffer-memory-utilization {
+                frequency 300s;
+				/*
+                * Sets color to red and sends out an anomaly notification when
+                * memory buffer utilization count increases more than threshold static ($threshold) value
+                * 
+                */
+                term is-buffer-memory-utilization-is-abnormal {
+                    when {
+                        greater-than "$memory-buffer-utilization" "$threshold";
+                    }
+                    then {                      
+                        status {
+                            color red;
+                            message "Buffer memory utilization for fpc slot=($slot) is crossed static threshold value ($threshold), Value=($memory-buffer-utilization)";
+                        }
+                    }
+                }
+				/*
+                * Sets color to yellow and sends out an anomaly notification when
+                * memory buffer utilization count increases more than dynamic threshold ($threshold) value
+                * 
+                */
+                term is-buffer-memory-utilization-is-minimal {
+                    when {
+                        equal-to "$dt-buffer-util" 1;
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "buffer memory utilization for fpc slot=($slot) is crossed dynamic threshold value, Value=($memory-buffer-utilization)";
+                        }
+                    }
+                }
+				/*
+                * Defaults color to green.
+                */
+                term is-buffer-memory-utilization-is-normal {
+                    then {
+                        status {
+                            color green;
+                            message "Buffer memory utilization for fpc slot=($slot) is normal, Value=($memory-buffer-utilization)";
+                        }
+                    }
+                }
+            }
+			/*
+            * Anomaly detection logic for CPU utilization.
+            */
+            trigger cpu-utilization {
+                frequency 300s;
+               	/*
+                * Sets color to red and sends out an anomaly notification when
+                * CPU utilization count increases more than static static ($threshold) value
+                * 
+                */				
+                term is-cpu-utilization-is-abnormal {

```

Comment:
Please fix double "is" for non-default terms - across all instances.

Reply:
Fixed